aip/general/0008.md

```diff
@@ -7,22 +7,40 @@ placement:
   order: 40
 ---
 
-# AIP Style guide
+# AIP Style and Guidance
 
 AIP stands for **API Improvement Proposal**, which is a design document
-providing high-level, concise documentation for API development. The goal is for
-these documents to serve as the source of truth for API-related documentation at
-Google and the way API teams discuss and come to consensus on API guidance.
+providing high-level, concise documentation for API design and development. The
+goal is for these documents to serve as the source of truth for API-related
+documentation at Google and the way API teams discuss and come to consensus on
+API guidance.
 
 AIPs are most useful when they are clear and concise, and cover a single topic
 or inquiry well. In the same way that AIPs describe consistent patterns and
 style for use in APIs, they also _follow_ consistent patterns and style.
 
 ## Guidance
 
-AIPs **must** cover a single, discrete topic, and **should** fundamentally
-answer the question, "What do I do?" with actionable guidance. AIPs **may** also
-cover what _not_ to do, but **should not** cover _only_ anti-patterns.
+- AIPs **must** cover a single, discrete topic, and provide clear, actionable
+  guidance.
+- AIPs **must not** duplicate or contradict guidance in another AIP.
+- AIPs **may** also cover what _not_ to do, but **should not** cover _only_
+  anti-patterns.
+- If AIP guidance is conditional (e.g. a design pattern such as Jobs), the
+  guidance **must** clearly explain under what conditions the guidance should be
+  followed.
+
+AIPs **must** contain API guidance that provide a clear benefit to one or more
+clients, including but not limited to:
+
+- Command line interfaces.
+- [Infrastructure as Code][] tools.
+- SDKs to interact with an API from a programming language.
+- Visual User Interfaces.
+- Users.
```

Comment:
IMO, "users" is vague and is what led to the previous style-guide approach.

Reply:
Users is indeed vague, and I think that if guidance was *only* suppose to benefit the user, then it leaves a lot of room for interpretation.

However, this section codifies that *in addition to* users, one must consider the clients as well. I think it's important to consider users in AIPs since there is guidance that's tangible to them (e.g. consistency in units and naming)